web/src/components/RemoteContentSandBox.ts

```diff
@@ -144,6 +144,38 @@ const RemoteContentSandBox = styled.div<{ $centered: boolean; $smallText: boolea
     font-weight: bold;
   }
 
+  .contact-card {
+    display: inline-block;
+    box-sizing: border-box;
+    padding: 1rem;
+    border-radius: 0.25rem;
+    background-repeat: no-repeat;
+    background-color: rgb(127 127 127 / 15%) !important;
+    background-image: linear-gradient(to right, rgb(255 255 255 / 90%) 0 100%),
+      url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij48cGF0aCBkPSJNMTMuMDE4IDEyLjQ4aC0yLjAzNkE5LjA5IDkuMDkgMCAwIDAgMS45MiAyMS42YS40OC40OCAwIDAgMCAuNDguNDhoMTkuMmEuNTMuNTMgMCAwIDAgLjQ4LS41MzggOS4wOCA5LjA4IDAgMCAwLTkuMDYyLTkuMDYyTTE2LjggNi43MmE0LjggNC44IDAgMCAxLTQuOCA0LjggNC44IDQuOCAwIDAgMS00LjgtNC44IDQuOCA0LjggMCAwIDEgNC44LTQuOCA0LjggNC44IDAgMCAxIDQuOCA0LjgiLz48L3N2Zz4=') !important;
```

Comment:
:question: Is there a way to load an image from our assets instead of hardcoding it here in bae64? I think that would improve maintainability and readability of the code. No idea though, so if you say that does not work its fine as well.

Reply:
Yes, we're doing it in this file for the class "link-external", a little higher up.

Reply:
Yeah I was about to say that I can't use the imported image in native as a background url